packages/apis/ipfs/src/query/schema.graphql

```diff
@@ -0,0 +1,6 @@
+#import { Query, ResponseType } into Http from "w3://ens/http.web3api.eth"
+
+type Query {
+  catToString(ipfsUrl: String!, cid: String!): String!
+  catFile(ipfsUrl: String!, cid: String!): Bytes!
```

Comment:
Just FYI, in the future when Web3API ENV Variables are supported, the `ipfsUrl` argument will become optional. If it isn't specified in the method's input arguments, the Web3API will try to get it from the ENV. If neither exist, an error occurs.